src/Main.java

```diff
@@ -1,65 +1,19 @@
-import java.util.InputMismatchException;
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class Main {
 
     public static void main(String[] args) {
 
         UsersBook usersBook = new UsersBook();
+        CommandInfoList commandInfoList = new CommandInfoList();
+        Commands commands = new Commands(usersBook);
         Scanner scanner = new Scanner(System.in);
-//      boolean saveToDisk = true;
 
         while (true) {
-            System.out.println("Чтобы получить данные всех пользователей, введите команду => 'getAllUsers'");
-            System.out.println("Чтобы получить данные пользователя по ID , введите команду => 'getUserForId'");
-            System.out.println("Чтобы добавить нового пользователя, введи команду => 'add'");
-
+            commandInfoList.printCommandInfoList(new ArrayList<>(commandInfoList.initialization()));
```

Comment:
You just need to print all commands to user. What if System.out.println(commands) will do the same ? Then you can get rid of CommandInfoList